my-app/src/constants/Game.jsx

```diff
@@ -0,0 +1,21 @@
+export const urlSendData = 'http://localhost:8080/api/aiProcessor';
+export const urlSendRole = 'http://localhost:8080/api/aiRole';
+
+export const SQUARE_CLICKED = 'SQUARE_CLICKED';
+export const BEFORE_SEE_HISTORY = 'BEFORE_SEE_HISTORY';
+export const SWITCH_PLAYER = 'SWITCH_PLAYER';
+export const PLAYER_ROLE = 'PLAYER_ROLE';
+export const SET_END_GAME = 'SET_END_GAME';
+export const CLEAR_HISTORY = 'CLEAR_HISTORY';
+export const SET_STORE = 'SET_STORE';
+export const SET_VISIBLE_ROLE_OPTIONS = 'SET_VISIBLE_ROLE_OPTIONS';
```

Comment:
let's make two actions out of one:
`SHOW_ROLE_OPTIONS`
`HIDE_ROLE_OPTIONS`
you will get each in your reducer function, and set the corresponding boolean value to the `isRoleOptionsVisible` field